src/utils/LaunchUtils.ts

```diff
@@ -0,0 +1,70 @@
+import Profile from "../model/Profile";
+import R2Error from "../model/errors/R2Error";
+import Game from "../model/game/Game";
+import FsProvider from "../providers/generic/file/FsProvider";
+import GameRunnerProvider from "../providers/generic/game/GameRunnerProvider";
+import GameDirectoryResolverProvider from "../providers/ror2/game/GameDirectoryResolverProvider";
+import ManagerSettings from "../r2mm/manager/ManagerSettings";
+import ModLinker from "../r2mm/manager/ModLinker";
+
+export const launch = async (game: Game, profile: Profile, modded: boolean): Promise<void> => {
```

Comment:
I'd recommend, with @MythicManiac to approve, that these are moved into a class and defined as public static methods.

The reasoning behind this is that when used, these look like they'd be defined within the same file as a method, whereas these are actually just imported. We can also tell the exact area where the action is coming from without having to refer back to the import statement.

Another useful thing is that when it's class based, we can likely mock it in tests pretty easily.